modules/manage/pages/iceberg/choose-iceberg-mode.adoc

```diff
@@ -0,0 +1,151 @@
+
+ifdef::env-cloud[]
+In the Schema Registry wire format, a "magic byte" and schema ID are embedded in the message payload header. Producers to the topic must use the wire format in the serialization process so Redpanda can determine the schema used for each record, use the schema to define the Iceberg table, and store the topic values in the corresponding table columns. See also: the https://www.redpanda.com/blog/schema-registry-kafka-streaming#how-does-serialization-work-with-schema-registry-in-kafka[Understanding Apache Kafka Schema Registry^] blog post.
+endif::[]
+
+=== value_schema_latest
+
+Creates an Iceberg table whose structure matches the latest schema registered for the subject in the Schema Registry. You must register a schema in the xref:manage:schema-reg/schema-reg-overview.adoc[Schema Registry]. Unlike the `value_schema_id_prefix` mode,  `value_schema_latest` does not require that producers use the wire format.
+
+=== disabled
+
+Default for `redpanda.iceberg.mode`. Disables writing to an Iceberg table for the topic.
+
+== Configure Iceberg mode for topic
+
+You can set the Iceberg mode for a topic when you create the topic, or you can update the mode for an existing topic.
+
+.Create a new topic and set `redpanda.iceberg.mode`:
+[,bash]
+----
+rpk topic create <topic-name> --topic-config=redpanda.iceberg.mode=<iceberg-mode>
+----
+
+.Set `redpanda.iceberg.mode` for an existing topic:
+[,bash]
+----
+rpk topic alter-config <new-topic-name> --set redpanda.iceberg.mode=<iceberg-mode>
+----
+
+[[override-value-schema-latest-default]]
+=== Override `value_schema_latest` default
+
+In `value_schema_latest` mode, only the string `value_schema_latest` is required in the property value. This sets `value_schema_latest` mode to its default behavior, which derives the subject for the topic using xref:manage:schema-reg/schema-id-validation.adoc#set-subject-name-strategy-per-topic[TopicNameStrategy]. For Protobuf data, the default behavior also deserializes records using the first message within the corresponding Protobuf schema in the Schema Registry.
+
+To override the default behavior, use the following optional syntax:
+
+[,bash]
+----
+value_schema_latest:subject=<subject-name>,protobuf_name=<protobuf-message-full-name>
+----
+
+* For both Avro and Protobuf, specify a different subject name by using the key-value pair `subject=<subject-name>`, for example `value_schema_latest:subject=sensor-data`.
+* For Protobuf only:
+** Specify a different message definition by using a key-value pair `protobuf_name=<message-name>`, for example `value_schema_latest:protobuf_name=com.example.manufacturing.SensorData`.
+** To specify both a different subject and message definition, separate the key-value pairs with a comma, for example `value_schema_latest:subject=my_protobuf_schema,protobuf_name=com.example.manufacturing.SensorData`.
+
+== How Iceberg modes translate to table format
```

Comment:
I feel this we should either link to or inline the content that describes the full conversion rules for protobuf + avro.

Reply:
@rockwotj Moved the entire section from About doc to a new subsection - https://github.com/redpanda-data/docs/pull/1068/files#diff-5076fb238fceeabc4117ed4246fd8815832779df3ff7f6eff2784d01d4fcdf6cR157 does moving it inline like this make sense?

Reply:
I like it! Thanks